tests/gui/test_main_window.py

```diff
@@ -15,10 +14,7 @@
 from dangerzone.gui.logic import DangerzoneGui
 from dangerzone.gui.main_window import ContentWidget
 from dangerzone.gui.updater import UpdateReport, UpdaterThread
-from dangerzone.util import get_version
 
-from .. import sample_doc, sample_pdf
-from . import qt_updater as updater
```

Comment:
I found it weird to have the fixtures imported (because they don't need to be), but I might be missing something?

Reply:
Taken care of in #813